modules/control/controller/mpc_controller.cc

```diff
@@ -365,6 +366,14 @@ Status MPCController::ComputeControlCommand(
   Matrix upper_bound(controls_, 1);
   upper_bound << wheel_single_direction_max_degree_, max_acceleration_;
 
+  double max = std::numeric_limits<double>::max();
+  Matrix lower_state_bound(basic_state_size_, 1);
+  lower_state_bound << -1.0 * max, -1.0 * max, -1.0 * max, -1.0 * max,
+      -1.0 * max, -1.0 * max;
+
+  Matrix upper_state_bound(basic_state_size_, 1);
```

Comment:
We may need to load the state confs from configs instead of just setting it to max, please add TODO here